power_spec.adoc

```diff
@@ -0,0 +1,161 @@
+submission type (e.g., a generally available system must use
+software/firmware qualified for general availability and shipping
+with the platform)
+* Include no benchmark or benchmarking-specific modifications
+* Changes in power management behavior must not have manual intervention or
+be aware of the benchmark
+
+== Wall-Powered System Under Test
+
+The System under Test (SUT) includes any component that contributes to the
+measured performance of the MLPerf benchmark. For example, an
+accelerator based system should also include the host node,
+networking/switches for distributed systems, number and type of cooling systems,
+and hard disks. If the system uses shared resources with multiple servers, the
+shared resources must be included in the SUT.
+
+The cooling method must be described (e.g., air-cooling, liquid cooling).
+
+Figure 1 below illustrates details of the MLPerf power measurement configuration.
+Note that in contrast to SPECpower, the temperature sensor is not required, and
+the collection system is not coupled to the SUT and driving/controlling the SUT.
+All power delivered to the SUT must arrive through the power meter.
+
+DC-powered systems are also supported with an appropriate power meter, but may
+not be compared to AC-powered systems. If a wall-powered system is available in
+both an AC-powered and DC-powered configuration, we require using the
+AC-powered configuration.
+
+image::sut_power_setup.png[SUT Setup]
+
+== Wall-Powered Measurement Procedure
+
+. Connect an approved power meter from this list to the SUT.
+
+ For single enclosure devices (where all the components are
+ contained inside a single chassis), connect the power supply
+ to the meter. For disaggregated systems, or systems with
+ multiple nodes, all power supplies for the SUT components
+ must be connected to the power meter. For logging, the
+ power meter should be connected to an external computer
+ (not in the SUT) which is running the SPEC PTDaemon
+ logging software. See Figure 1 above.
+
+ This approach applies to both AC and DC powered systems,
+ but the type of power delivery must be specified and AC
+ and DC systems cannot be compared to one another due to
+ differences in conversion efficiency.
+
+. Synchronize system clock of two systems with Network Time Protocol (NTP)
+
+ More details to be shared as the procedure is being piloted.
+
+. Perform pre-processing of inference data
+
+ As long as pre-processing of inference data is not measured for performance,
+ it should be performed ahead of the actual inferencing and not
+ measured from a power standpoint.
+
+. Pause the system for two minutes prior to start of the benchmark.
+
+ This allows the system to move to a lower power state after performing the
+ pre-processing and before running the benchmark.
+
+. Run the MLPerf Inference benchmark for a specific scenario and model
+
+ The system is running the benchmark and in the 'Active' state. Only a single
```

Comment:
This is the first time "Active" is used. It would be good to have a small definition section in this document.